worker-caf/src/main/java/com/hpe/caf/worker/AbstractWorker.java

```diff
@@ -130,9 +130,9 @@ protected final WorkerResponse createSuccessResult(final V result, final byte[]
      *
      * @return a WorkerResponse that represents a success
      */
-    protected final WorkerResponse createSuccessNoOutputToQueue()
+    protected final WorkerResponse createSuccessNoOutputToQueue(boolean resetTrackTo)
```

Comment:
This looks like a breaking change - any worker previously using `createSuccessNoOutputToQueue()` would presumably not work following this change.  Better to add a new method for this.